# barley_pangenes

Pangenes are clusters of gene models/alleles found in barley assemblies in a similar genomic location.

They are computed from Whole Genome Alignments and nucleotide sequences, not protein sequences.

Click on chromosomes below to browse pangenes and display multiple sequence alignments of encoded proteins. 
Read our [blog post](https://bioinfoperl.blogspot.com/2024/02/browsing-barley-pangenes.html) for more information.

[chr1H](./chr1H.html) | [chr2H](./chr2H.html) | [chr3H](./chr3H.html) | [chr4H](./chr4H.html) | [chr5H](./chr5H.html) | [chr6H](./chr6H.html) | [chr7H](./chr7H.html) | [chrUn](./chrUn.html) 

![example pangene](https://media.springernature.com/lw685/springer-static/image/art%3A10.1186%2Fs13059-023-03071-z/MediaObjects/13059_2023_3071_Fig4_HTML.png)

## Raw files

You can download the raw pangene matrices and get more details at <https://github.com/eead-csic-compbio/barley_pangenes>.

## Citation

Protocol and files first published at:

Contreras-Moreira B, Saraf S, Naamati G et al (2023) GET_PANGENES: calling pangenes from plant genome alignments confirms presence-absence variation. Genome Biol 24, 223. https://doi.org/10.1186/s13059-023-03071-z

Multiple sequence alignments produced with clustal-omega v1.2.4:

Sievers F, Wilm A, Dineen D, et al (2011) Fast, scalable generation of high-quality protein multiple sequence alignments using Clustal Omega. Mol Syst Biol 7:539. https://doi.org/10.1038%2Fmsb.2011.75

## Funding 

Multiple alignments of coding sequences from barley pangenes were computed within [PRIMA project GENDIBAR](https://www.era-learn.eu/network-information/networks/prima/section-2-call-multi-topic-2018/use-of-local-genetic-diversity-to-understand-and-exploit-barley-adaptation-to-harsh-environments-and-for-pre-breeding).
